app/fonts/icons/chatBubbles.svg

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generated by IcoMoon.io -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="54" height="48" viewBox="0 0 54 48">
-<path fill="#000000" d="M22.5 0v0c12.426 0 22.5 8.152 22.5 18.209s-10.074 18.209-22.5 18.209c-1.193 0-2.365-0.076-3.508-0.221-4.833 4.814-10.414 5.677-15.992 5.804v-1.178c3.012-1.47 5.438-4.147 5.438-7.206 0-0.427-0.033-0.846-0.095-1.256-5.089-3.338-8.342-8.437-8.342-14.151 0-10.056 10.074-18.209 22.5-18.209zM46.688 40.813c0 2.622 1.702 4.917 4.313 6.177v1.010c-4.834-0.109-9.271-0.849-13.46-4.975-0.991 0.124-2.006 0.189-3.040 0.189-4.475 0-8.597-1.207-11.888-3.236 6.782-0.024 13.184-2.196 18.039-6.125 2.447-1.981 4.378-4.304 5.738-6.907 1.442-2.759 2.173-5.699 2.173-8.738 0-0.49-0.021-0.977-0.059-1.461 3.402 2.809 5.496 6.639 5.496 10.86 0 4.898-2.82 9.268-7.23 12.13-0.054 0.351-0.082 0.711-0.082 1.077z"></path>
-</svg>
```

Comment:
getting rid of this - we only use it in our internal build and it can be replaced there with a font-awesome alternative